include/deal.II/trilinos/nox.h

```diff
@@ -49,16 +49,16 @@ namespace TrilinosWrappers
    * // create nonlinear solver
    * TrilinosWrappers::NOXSolver<VectorType> solver(additional_data);
    *
-   * // set user functions to compute residual,
-   * // set up Jacobian, and to apply the inverse of
-   * // Jacobian; note that there are more functions that
-   * // can be set
+   * // Set user functions to compute residual,
+   * // set up the Jacobian, and to apply the inverse of
```

Comment:
Shouldn't we use
```suggestion
   * // to set up the Jacobian, and to apply the inverse of
```
for consistent use of prepositions within the sentence?

Reply:
Yes, you're correct. I've fixed both errors, and have adjusted the line breaks for this set of comments.